app/services/user_validation_service.rb

```diff
@@ -126,8 +125,7 @@ def validate_user_pre_auth_checks(user)
     return ValidateResult.new(:fail, _("Error: New password and verify password must be the same")) if
       user[:new_password].present? && user[:new_password] != user[:verify_password]
 
-    return ValidateResult.new(:fail, _("Error: New password can not be blank")) if
-      user[:new_password] && user[:new_password].blank?
+    return ValidateResult.new(:fail, _("Error: New password can not be blank")) if user[:new_password]&.blank?
```

Comment:
Potentially wrong transformation.

`false && false.blank?` is false, but `false&.blank?` is true

Reply:
OK, reverted.